Description: Switch to deflate-zlib for zip crate as deflate-zlib-rs is disabled in Debian
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,12 +46,11 @@ repository = "https://github.com/mozilla
 
 [features]
 default = [
-    "deflate-zlib-rs",
+    "deflate-zlib",
     "demangle-no-swift",
 ]
 deflate = ["zip/deflate"]
 deflate-zlib = ["zip/deflate-flate2-zlib"]
-deflate-zlib-rs = ["zip/deflate-flate2-zlib-rs"]
 demangle-no-swift = [
     "symbolic-demangle/cpp",
 #    "symbolic-demangle/msvc",
